CV/#sourse/scss/style.scss

```
@@ -0,0 +1,303 @@
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
+*,
+*:before,
+*:after {
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+:focus,
+:active {
+  outline: none;
+}
+a:focus,
+a:active {
+  outline: none;
+}
+nav,
+footer,
+header,
+aside {
+  display: block;
+}
+html,
+body {
+  height: 100%;
+  width: 100%;
+  font-size: 100%;
+  line-height: 1;
+  font-size: 14px;
+  -ms-text-size-adjust: 100%;
+  -moz-text-size-adjust: 100%;
+}
+input,
+button,
+textarea {
+  font-family: inherit;
+}
+input::-ms-clear {
+  display: block;
+}
+button {
+  cursor: pointer;
+}
+button::-moz-focus-inner {
+  padding: 0;
+  border: 0;
+}
+
+a:visited {
+  text-decoration: none;
+}
+a:hover {
+  text-decoration: none;
+}
+ul li {
+  list-style: none;
+}
+img {
+  vertical-align: top;
+}
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-size: inherit;
+  font-weight: 400;
+}
+
+body {
+  font-family: roboto;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 1.4;
+}
+
+.container {
+  max-width: 1280px;
+  margin: 0 auto;
+  padding: 0 20px;
+}
+
+.header {
+  background-color: cadetblue;
+}
+.nav__list {
+  height: 70px;
+
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  a {
+    text-decoration: none;
+    color: white;
+    font-size: 18px;
+    font-weight: 500;
+    transition: 0.5s all;
+  }
+  a:hover {
+    color: black;
+  }
+}
+@media (max-width: 820px) {
+  .header {
+    display: none;
+  }
+  main {
+    padding: 10px 0 0 0;
+  }
+}
+.markup {
+  display: flex;
+  .markup__left {
```

Если использовать БЭМ с SCSS, то "&" помогает убрать дупликацию:
```suggestion
  & &__left {
```
или
```suggestion
  &__left {
```